File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/fragments/drag_and_drop/model/Item.java
package de.dimedis.mobileentry.fragments.drag_and_drop.model;

public class Item {
    public String functionName;
    public String ItemString;
    private int iconRes;

    public Item() {
    }

    public Item(String t, int iconResId, String functionName) {
        ItemString = t;
        this.functionName = functionName;
        iconRes = iconResId;
    }

    public String getItemString() {
        return ItemString;
    }

    public int getIconRes() {
        return iconRes;
    }

    protected int position;

    public void setPosition(int position) {
        this.position = position;
    }

    public int getPosition() {
        return position;
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/model/Item.kt
package de.dimedis.mobileentry.model

open class Item {
    var functionName: String? = null
    var ItemString: String? = null
    var iconRes = 0

    constructor() {}
    constructor(t: String?, iconResId: Int, functionName: String?) {
        ItemString = t
        this.functionName = functionName
        iconRes = iconResId
    }

    val getItemString: String?
        get() = ItemString

    val getIconRes: Int
        get() = iconRes

    var position = 0

    @JvmName("setPosition1")
    fun setPosition(position: Int) {
        this.position = position
    }

    val getPosition: Int
        get() = position
}